### PR TITLE
Adding May 2019 webinar to events

### PR DESCRIPTION
**Pull request by kevinbluer**

### CODE REVIEW

File: src/events/data.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "title": "The Future of Blockchain Solution Development",
+    "start": "Weds, May 22, 2019, 3:00 PM EST",
```
Comment (seesemichaelj) [inline src/events/data.json]: we're in EDT not EST

File: src/events/data.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    "end": "3:45 PM EST",
```
Comment (seesemichaelj) [inline src/events/data.json]: same here

File: src/events/webinar-truffle-future.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# THE FUTURE OF BLOCKCHAIN SOLUTION DEVELOPMENT
+
+
+<div class="row">
+  <div class="col-sm-6 mt-4">
+    <h3 class="mb-3">Overview</h3>
+    <p>In this upcoming webinar Tim Coulter and Kevin Bluer explore the blockchain solution development landscape, specifically looking at the tools, development workflows / lifecycles, and emerging best practices. They will also review a number of the upcoming features in the Truffle Suite itself, all designed to make your development experience all the sweeter (and efficient). Note that this session assumes some pre-existing development knowledge and prior experience with Truffle.</p>
+    <h3 class="mt-3 mb-3">Details</h3>
+    <p class="mb-0"><strong>Date: </strong> May 22nd, 2019</p>
+    <p class="mt-0"><strong>Time: </strong> 3:00-3:45 PM EST</p>
```
Comment (seesemichaelj) [inline src/events/webinar-truffle-future.md]: another EST => EDT

File: src/events/data.json (hunk 1 above)
Comment (DiscRiskandBisque) [reply src/events/data.json]: I've started omitting the daylight/standard designation when spelling out times for this reason. Perhaps change them to "ET"?

**Comment by DiscRiskandBisque** [comment]
Generalized the timezones with `EST` changing to `ET`. 👽

**Pull request merged**